modules/application-platform/src/main/java/org/testcontainers/containers/ApplicationContainer.java

```diff
@@ -0,0 +1,331 @@
+package org.testcontainers.containers;
```

Comment:
If the module suggestion is ok then we need to rename the package to `org.testcontainers.applicationserver`